packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -693,6 +693,10 @@ export class Combobox
           }
           event.preventDefault();
         }
+
+        if ((event.target as HTMLElement).closest("calcite-chip")) {
+          event.stopPropagation();
```

Comment:
Can you use event cancelation instead? Per [our guidelines](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/README.md#privateinternal-events), we don't want to stop propagation of _public_ events.